browse/services/listing/__init__.py

```diff
@@ -28,10 +29,10 @@ def fs_listing(config: dict, _: Any) -> "ListingService":
     return FsListingFilesService(config["DOCUMENT_LISTING_PATH"])
 
 def db_listing(config: dict, _: Any) -> "ListingService":
-    """Factory function for filesystem-based listing service."""
-    from browse.services.database import models
-    from .db_listing import DBListingService
-    return DBListingService(models.db)
+    """Factory function for DB backed listing service."""
+    from .db_listings import DBListingService
+    # maybe pass in the specific classes for the tables we need?
+    return DBListingService()
```

Comment:
did you test using the db listing service without it being passed anything?

Reply:
Yup!

Reply:
sweet